### ISSUE TITLE
About warning "Font shape undefined"

### ISSUE THREAD

**Issue by FantasqueX** [zhaofeng-shu33/THU-coursework-template#4]
Hi, there! I really appreciate your work, and this project makes my life easier.

But I met a small problem that when I build the ithw.tex with XeLaTeX, I got the warning "Font shape `TU/SimSun(0)/m/sc' undefined(Font) using `TU/SimSun(0)/m/n' instead". I want to know how can I remove the warning, thank you!

**Comment by zhaofeng-shu33** [comment]
`ithw.tex` does not have any chinese characters. You can use `pdflatex` to compile it. I think it will not have
the warning if you change your compiler.

**Comment by FantasqueX** [comment]
I try to use PDFLaTeX to compile` ithw.tex ` , but the warning is still there like this "Font shape `C70/rm/m/sc' undefined(Font) using `C70/rm/m/n' instead".

**Comment by zhaofeng-shu33** [comment]
Have you installed the full version of texlive or any other equivalent platform distribution? I guess there are
some missing fonts. Once you installed them, there will not be any warning.

**Comment by FantasqueX** [comment]
I use MikTeX on Windows. And I only install the package which the document requires. So which package should I install then?
And I'll try to install a full version of MikTeX.

**Comment by zhaofeng-shu33** [comment]
> `ithw.tex` does not have any chinese characters. You can use `pdflatex` to compile it. I think it will not have
> the warning if you change your compiler.

Sorry, I mean `iihw.tex` does not have any chinese characters.

**Comment by FantasqueX** [comment]
When I used PDFLaTeX on ArchLinux with package texlive-full and texlive-langchinese, I got a error message "! Critical Package ctex Error: CTeX fontset `fandol' is unavailable in current(ctex) mode.Type <return> to continue.... }". When I used XeLaTeX, I got a warning message "Font shape `TU/FandolSong-Regular(0)/m/sc' undefined(Font) using `TU/FandolSong-Regular(0)/m/n' instead".
I think the problem is the lack of fonts.
